test/unit/views/MessagesStub.scala

```diff
@@ -31,6 +31,9 @@ trait MessagesStub {
     Fails the test if a view is configured with a message key that doesnt exist in the messages file
    */
   private class AllMessageKeysAreMandatoryMessages(msg: Messages) extends Messages {
+
+    override def asJava: play.i18n.Messages = msg.asJava
```

Comment:
Same as previous comment.

Reply:
I added it (and the same above in ErrorHandlerSpec) to resolve this build error - `class AllMessageKeysAreMandatoryMessages needs to be abstract, since method asJava in trait Messages of type => play.i18n.Messages is not defined`

Reply:
You can change it this way
```
  override def asJava: play.i18n.Messages = new play.i18n.MessagesImpl(lang.asJava, messagesApi.asJava)
```